src/main/java/org/hvdw/jexiftoolgui/datetime/ShiftDateTime.java

```diff
@@ -71,14 +60,10 @@ private void initDialog() {
     }
 
     private void writeInfo() {
-        String fpath = "";
-        String shiftOption = "";
-        String res = "";
-        List<String> cmdparams = new ArrayList<String>();
```

Comment:
ArrayList without the definition of its size and without the need to get specific indexes are expensive. LinkedLists are cheaper.